src/main/java/duke/commands/Command.java

```diff
@@ -0,0 +1,162 @@
+package duke.commands;
+
+import duke.tasks.*;
```

Comment:
Try to import all classes used, instead of *. 
In IntelliJ, the * import can be disabled by going to 
Settings -> Editor -> Code Style -> Java, and under the **Imports** tab, the number of class count before * is used can be increased. (Under `Class count to use import with *`). I've changed mine to a reasonably higher number (200). 

![image](https://user-images.githubusercontent.com/38811217/73627389-7d017c00-4687-11ea-860e-c2ac09f1ad05.png)